src/main/java/uk/gov/ons/ssdc/caseprocessor/utils/SampleValidateHelper.java

```diff
@@ -13,10 +13,10 @@ public static void validateNewValue(
     if (columnValidator.getColumnName().equals(entry.getKey())) {
       Map<String, String> validateThis = Map.of(entry.getKey(), entry.getValue());
 
-      Optional<String> validationErrors = columnValidator.validateRow(validateThis);
+      Optional<String> validationErrors =
+          columnValidator.validateRowWithDataExcludedErrorMsgs(validateThis);
```

Comment:
this code seems over-complex, seeing as we just want to validate a single value against a single column's validators